source/fuzz/shrinker.cpp

```diff
@@ -180,7 +180,7 @@ Shrinker::ShrinkerResultStatus Shrinker::Run(
       // transformations inapplicable.
       std::vector<uint32_t> next_binary;
       protobufs::TransformationSequence next_transformation_sequence;
-      if (Replayer(impl_->target_env)
+      if (Replayer(impl_->target_env, false)
```

Comment:
Should we allow validation in the shrinker too? Can you make the option affect the shrinker?

Reply:
Yes - good idea, done.

I have also enabled this in the replayer and shrinker tests (might make them take too long though).